examples/resources/stackit_network/resource.tf

```diff
@@ -1,9 +1,12 @@
 resource "stackit_network" "example" {
   project_id         = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
   name               = "example-network"
-  nameservers        = ["1.2.3.4", "5.6.7.8"]
+  ipv4_nameservers   = ["1.2.3.4", "5.6.7.8"]
   ipv4_prefix_length = 24
+  ipv4_gateway       = "10.1.2.1"
+  ipv4_prefix        = "10.1.2.0/24"
   labels = {
     "key" = "value"
   }
-}
+  routed = false
```

Comment:
Not sure if the example should set routed to false. We could confirm with IaaS

Reply:
Also, it currently errors when creating a networking without specifying `routed` or when setting it to `true`. This should be possible as it's the default behaviour

Reply:
as discussed I have implemented the `no_gateway` field checks

Reply:
also implemented no_ipv6_gateway field :)